netmiko/nokia/nokia_sros_ssh.py

```diff
@@ -279,10 +279,11 @@ def remote_file_size(self, remote_cmd=None, remote_file=None):
         if "File Not Found" in remote_out:
             raise IOError("Unable to find file on remote system")
 
+        dest_file_name = remote_file.replace("\\", "/").split("/")[-1]
```

Comment:
why this replacement is needed? what use case does it cover?

Reply:
Replace is indeed redundant, can be done as:
dest_file_name = remote_file.split("\\")[-1]
This is needed to separate filename from the path.

Reply:
Just talked to @jkierklo.

In SROS path can be specified with \ and with / they both will work.
This replace is needed to uniform regardless of what file/path is given split is done always on the same character.

Reply:
Users can specify file path as:
7705-TiMOS-8.0.R7**/**BOOT.LDR 
```A:LAB-SAR-8-2# file dir cf3:\7705-TiMOS-8.0.R7/BOOT.LDR 


Volume in drive cf3 on slot A has no label.

Volume in drive cf3 on slot A is formatted as FAT32

Directory of cf3:\7705-TiMOS-8.0.R7

09/26/2017  08:47p             2796640 BOOT.LDR
               1 File(s)                2796640 bytes.
               0 Dir(s)              1820561408 bytes free.```

Or like
7705-TiMOS-8.0.R7**\**BOOT.LDR 

```A:LAB-SAR-8-2# file dir cf3:\7705-TiMOS-8.0.R7\BOOT.LDR 


Volume in drive cf3 on slot A has no label.

Volume in drive cf3 on slot A is formatted as FAT32

Directory of cf3:\7705-TiMOS-8.0.R7

09/26/2017  08:47p             2796640 BOOT.LDR
               1 File(s)                2796640 bytes.
               0 Dir(s)              1820561408 bytes free.```

Reply:
yes, but why double backslashes? If, imagine, my path is `dir1\dir2\file` what will `\\` substitute?

Reply:
That's python ... first backslash is escape for the second.
```
>>>x = 'cf3:\dir\file'
>>> x
>>>'cf3:\\dir\x0cile'
>>>x = 'cf3:\\dir\\file'
>>>x
'cf3:\\dir\\file'
x.split('\\')
['cf3:', 'dir', 'file']
```

Reply:
ah, ok =) makes sense then

Reply:
I asked @jkierklo to make line 251 back to / and we all should be good.